src/utils/helpers.ts

```diff
@@ -0,0 +1,34 @@
+import { Errors } from '../types/Errors';
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+
+export const filterTodos = (todos: Todo[], filterType: FilterType) => {
+  return todos.filter(todo => {
+    switch (filterType) {
+      case FilterType.All:
+        return true;
```

Comment:
It is not necessary to indicate this case, because you are returning the `true` by `default`